src/Service/Models/DispatchedEmail.cs

```diff
@@ -0,0 +1,14 @@
+namespace Passwordless.Service.Models;
+
+public class DispatchedEmail : PerTenant
+{
+    public required Guid Id { get; set; }
+
+    public required DateTimeOffset CreatedAt { get; set; }
+
+    public required string UserId { get; set; }
+
+    public required string EmailAddress { get; set; }
```

Comment:
thought: We should make UserId and EmailAdress optional in the app, make it opt-out.

issue: We probably want to track the outgoing domain/url for monitoring missbehaviour.

Reply:
Technically, I only need the date here. I thought the additional fields would be useful if we need to identify the email. Should I remove them?

Reply:
@Tyrrrz We can keep them for now, but definitely add UrlTemplate here (without the tokens)

Reply:
RIP migrations again 😂

Reply:
Done